drivers/src/nrfx_nfct.c

```diff
@@ -556,6 +556,37 @@ nrfx_err_t nrfx_nfct_tx(nrfx_nfct_data_desc_t const * p_tx_data,
     return err;
 }
 
+nrfx_err_t nrfx_nfct_tx_bits(nrfx_nfct_data_desc_t const * p_tx_data,
```

Comment:
Please add some additional logic in this function like it is done here:
https://github.com/NordicSemiconductor/nrfx/blob/v2.7.0/drivers/src/nrfx_nfct.c#L523

Reply:
Good find. This is based on an older branch originally I think. I will do an update.

Reply:
Did a fix.